package.json

```diff
@@ -19,7 +19,8 @@
     "npm": ">=3"
   },
   "dependencies": {
-    "axios": "^0.21.4",
+    "axios": "0.21.4",
+    "chart.js": "^3.5.1",
```

Comment:
you need both chart.js and react-chartjs-2 ?

Reply:
As I understand react-chartjs-2 is just react wrapper for chart.js.

Reply:
This lib author wrote here https://www.npmjs.com/package/react-chartjs-2 about it.